### PR TITLE
Fix contribtest

### LINKED ISSUE #2453: [Bug]: Invalid workflow File ContribTests
### Describe the bug

> Check failure on line 52 in .github/workflows/contrib-tests.yml
GitHub Actions
/ ContribTests
Invalid workflow file
The workflow is not valid. .github/workflows/contrib-tests.yml (Line: 52, Col: 9): Unexpected value 'runs-on'

the contrib-tests workflow is failing due to the wrong usage of `runs-on` keyword in `.github/workflows/contrib-tests.yml (Line: 52, Col: 9)`.

cc @sonichi @ekzhu 

### Steps to reproduce

_No response_

### Model Used

_No response_

### Expected Behavior

_No response_

### Screenshots and logs

image

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by Hk669**
## Related issue number

Closes #2453 

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: autogen/agentchat/contrib/qdrant_retrieve_user_proxy_agent.py [hunk 1]
```diff
@@ -150,21 +150,21 @@ def retrieve_docs(self, problem: str, n_results: int = 20, search_string: str =
...
-    chunk_mode: str = "multi_lines",
-    must_break_at_empty_line: bool = True,
-    embedding_model: str = "BAAI/bge-small-en-v1.5",
-    custom_text_split_function: Callable = None,
-    custom_text_types: List[str] = TEXT_FORMATS,
-    recursive: bool = True,
-    extra_docs: bool = False,
-    parallel: int = 0,
-    on_disk: bool = False,
+    max_tokens: Optional[int] = 4000,
```
Comment (ekzhu) [inline autogen/agentchat/contrib/qdrant_retrieve_user_proxy_agent.py:153]: This doesn't seem to be part of the PR?

Comment (Hk669) [reply autogen/agentchat/contrib/qdrant_retrieve_user_proxy_agent.py:153]: yeah I don't know why this PR is combining the 2 branches into a single PR.

Comment (Hk669) [reply autogen/agentchat/contrib/qdrant_retrieve_user_proxy_agent.py:153]: ive mistakenly used the other branch as the base branch.
